content/docs/import/import-from-postgres.md

```diff
@@ -112,7 +112,7 @@ pg_dump -Fc -v -d <source_database_connection_string> | pg_restore -v -d <neon-d
 
 </CodeBlock>
 
-Piping is not recommended for medium (> 1 GB) and large databases (> 5 GB), as it is susceptible to failures during lengthy migration operations.
+Piping is not recommended for medium (> 1 GB) and large databases (> 5 GiB), as it is susceptible to failures during lengthy migration operations.
```

Comment:
FWIW i don't think it's critical to have GiB here.

up to docs team to decide

Reply:
Agree.